Utilities/DataFlow/test/test_SubframeUtils01.cxx

```diff
@@ -0,0 +1,56 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
```

Comment:
I can (and will) move to BOOST test, however I disagree on what you say:

- Tests themselves should not be compiled with `-DNDEBUG`, ever. If we do, we should fix the build rules.
- As noted elsewhere we should have the option to have two copy for each test: one compiled against the optimised library, one compiled with the full debug mode library.
- We should probably have a decision about which unit test framework to use, and not simply rely on consistency. FairRoot uses gtest (which is slightly nicer to read, at least for me, and does not depend on boost, which is always good for compile time) and I've heard good things about [catch](https://github.com/philsquared/Catch). @Barthelemy / @dberzano was there any evaluation of the framework for tests?  

@dberzano to comment, since this is really a tools issue.